script/constants/hostnames.js

```diff
@@ -1,10 +1,10 @@
 const environments = require('./environments');
 
 module.exports = {
-  [environments.DEVELOPMENT]: 'https://dev.vets.gov',
-  [environments.PREVIEW]: 'https://preview.va.gov',
-  [environments.PRODUCTION]: 'https://www.vets.gov',
-  [environments.STAGING]: 'https://staging.vets.gov',
-  [environments.VAGOVDEV]: 'https://dev.va.gov',
-  [environments.VAGOVSTAGING]: 'https://staging.va.gov',
+  [environments.DEVELOPMENT]: 'dev.vets.gov',
+  [environments.PREVIEW]: 'preview.va.gov',
+  [environments.PRODUCTION]: 'www.vets.gov',
+  [environments.STAGING]: 'staging.vets.gov',
+  [environments.VAGOVDEV]: 'dev.va.gov',
+  [environments.VAGOVSTAGING]: 'staging.va.gov',
 };
```

Comment:
This in particular was where I discovered the difference between host vs. hostname in Node vs. the browser. It makes sense to me to remove the protocol here and derive it as part of `hostUrl` instead, because it reconciles the format to match that of the `host` command line argument.